testing/src/main/java/ai/djl/testing/CoverageUtils.java

```diff
@@ -72,8 +72,8 @@ public static void testGetterSetters(Class<?> baseClass)
                 continue;
             }
 
-            Method[] methods = clazz.getDeclaredMethods();
-            for (Method method : methods) {
+            Method[] declaredMethods = clazz.getDeclaredMethods();
```

Comment:
`declaredMethods` might be more clear, but I don't feel `methods` is a bad local variable name.